### PR TITLE
Add ri

### PR DESCRIPTION
**Pull request by RJMW**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by RJMW** [comment]
@conda-forge/core - Please, any suggestions on how to fix the Windows build.

File: recipes/ri/bld.bat [hunk 1]
```diff
@@ -0,0 +1,8 @@
+mv "%RECIPE_DIR%\sys" "%SRC_DIR%\sys" 
+sed -i '66s/.*/#include \"sys\/time.h\"/' include\timer.h
```
Comment (scopatz) [inline recipes/ri/bld.bat:2]: you probably don't have sed on windows

Comment (RJMW) [reply recipes/ri/bld.bat]: @scopatz - I have replaced the sed command in the .bat file with a patch (e617868).

**Comment by scopatz** [comment]
You are getting an error in compilation: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=192766&view=logs&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=26d9c833-f81c-502c-87bc-0cb0be44f838&l=792

**Comment by RJMW** [comment]
> You are getting an error in compilation: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=192766&view=logs&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=26d9c833-f81c-502c-87bc-0cb0be44f838&l=792

@scopatz - I have fixed this. The current error is identical to the error we got before I switched from sed to patch.

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=192811&view=logs&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=26d9c833-f81c-502c-87bc-0cb0be44f838&l=804

**Comment by scopatz** [comment]
Yeah, I am not sure.  You could skip windows for now and re-enable it in the feedstock later? It seems like this project might not build on windows (or at least not using make?)

**Comment by RJMW** [comment]
@scopatz I have simplified the recipe to unix only. Please, can you check if any additional changes are required.

File: recipes/ri/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  skip: True  # [win]
+
+requirements:
+  build:
+    - make
+    - {{ compiler('cxx') }}
+
+test:
+  commands:
+    - which ri36
```
Comment (scopatz) [inline recipes/ri/meta.yaml:24]: We usually use `test -f ${PREFIX}/bin/ri36` here

Comment (RJMW) [reply recipes/ri/meta.yaml]: Done

**Comment by scopatz** [comment]
@conda-forge/staged-recipes - is anyone else seeing the following on Mac?

```
+ setup_conda_rc ./ ./recipes ./.ci_support/osx64.yaml
Configuring conda.
Traceback (most recent call last):
  File "/usr/local/miniconda/bin/setup_conda_rc", line 7, in <module>
    from conda_forge_ci_setup.build_utils import setup_conda_rc
ModuleNotFoundError: No module named 'conda_forge_ci_setup'
```

Also @RJMW - try merging from master

**Comment by RJMW** [comment]
@conda-forge-admin, please rerender

**Comment by RJMW** [comment]
@scopatz is this ready to merge?

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**